pkg/virt-handler/domain.go

```diff
@@ -36,11 +38,10 @@ type DomainDispatch struct {
 }
 
 func (d *DomainDispatch) Execute(indexer cache.Store, queue workqueue.RateLimitingInterface, key interface{}) {
-	obj, exists, err := indexer.GetByKey(key.(string))
-	if err != nil {
-		queue.AddRateLimited(key)
-		return
-	}
+	obj, exists, _ := indexer.GetByKey(key.(string))
+	// GetByKey can never return anything but nil for the error
```

Comment:
I think that, if I add it back in, we'll still be under the threshold for untestable code.  Let's see.